build-system/tasks/dist.js

```diff
@@ -452,4 +452,5 @@ dist.flags = {
   custom_version_mark: '  Set final digit (0-9) on auto-generated version',
   watch: '  Watches for changes in files, re-compiles when detected',
   closure_concurrency: '  Sets the number of concurrent invocations of closure',
+  defineExperimentConstant: '  Use to set constants for experiment builds',
```

Comment:
should this follow the naming convention (ie. snake_case) of the other keys in this object?

Reply:
@rcebulko I can change it if you think that's the way to go. I didn't add this flag, it had been in use before but was not working because it was not documented here. I don't have the context.

Reply:
I don't have familiarity with this part of the codebase, I'm going to remove myself and add `@rsimha` as he's most knowledgeable about the conventions here.

Reply:
Thanks for documenting the undocumented flag, @kevinkimball. Agree with @rcebulko, let's rename the flag to follow the naming convention.